cypress/views/bucket-policy.ts

```diff
@@ -0,0 +1,64 @@
+import { DATA_FEDERATION_NAMESPACE } from '../constants/common';
+import { Providers, TEST_DATA_SOURCE } from '../constants/tests';
+import { createDataSource } from './data-resource';
+
+export const createBucket = (
```

Comment:
Please make this as a keep value pair as we have for:
`export const MCGMSCommon = {
  visitMcgMsDashboard: () => {
	....
  },
  visitBucketPolicyList: () => {
  	.....
  },
  visitDataSourceListPage: () => {
	....
  },
};`

Reply:
can be named as BPCommons